test/replica-isolation/mzcompose.py

```diff
@@ -72,7 +72,10 @@ def drop_create_replica(c: Composition) -> None:
         dedent(
             """
             > DROP CLUSTER REPLICA cluster1.replica1
-            > CREATE CLUSTER REPLICA cluster1.replica3 REMOTE ['computed_1_1:2100', 'computed_1_2:2100']
+            > CREATE CLUSTER REPLICA cluster1.replica3
+              REMOTE ['computed_1_1:2100', 'computed_1_2:2100'],
+              COMPUTE ['computed_1_1:2102', 'computed_1_2:2102'],
+              WORKERS 1
```

Comment:
Can we omit all the `WORKERS 1` defaults from the mzcompose.py files and use a `1` default? The reason is that in the future I will need to introduce a default workers size setting (see https://github.com/MaterializeInc/materialize/pull/14272#issuecomment-1220356095) and all those `1` defaults interspersed throughout will be constantly overriding it.

Reply:
Yes, I think this would make sense!